client/src/types/internal.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  AxiomV2CircuitCapacity,
+} from "@axiom-crypto/circuit";
+import { 
+  AxiomV2CallbackInput,
+  AxiomV2CompiledCircuit,
+  AxiomV2QueryOptions,
+  ChainConfig,
+  ClientConfig,
+  BridgeType,
+} from "./external";
+
+export interface CoreConfig {
+  compiledCircuit: AxiomV2CompiledCircuit;
+  callback: AxiomV2CallbackInput;
```

Comment:
ah sorry; I know we discussed this previously, but after some consideration, the main issue when combining it into a single file is that we actually need to still export the types from the file so that they are available internally within the package itself. however, we want to have a clean export for types that the user should see versus the ones that the user should not. 

the other way to do it is to, in `types/index.ts`, do something like:
```
export { each, type, name } from "./external"
```
but this is a bit fragile in that we need to manually add the type in two separate files to have it exported to the user.

Reply:
Ah, I see -- that makes sense, we can keep this.